pandas/tests/copy_view/test_replace.py

```diff
@@ -36,3 +37,22 @@ def test_replace_inplace_reference(using_copy_on_write):
         assert view._mgr._has_no_reference(0)
     else:
         assert np.shares_memory(get_array(df, "a"), arr_a)
+
+
+@pytest.mark.parametrize("method", ["where", "mask"])
+def test_masking_inplace(using_copy_on_write, method):
+    df = DataFrame({"a": [1.5, 2, 3]})
+    df_orig = df.copy()
+    arr_a = get_array(df, "a")
+    view = df[:]  # noqa
```

Comment:
You can remove this (might have forgotten this as well), you are using view again a bit further down

Reply:
Thanks for the catch, updated your tests as well.